### PR TITLE
Feature/suggestion search

### PR DESCRIPTION
**Pull request by rachadiannovansyah**
Overview:
- Add elastic suggestion search API

@jabardigitalservice/jds-backend 

Evidence:
project: Portal Jabar
title: Add API suggestion search elastic
participants: @rachadiannovansyah @khihadysucahyo @firmanJS @ayocodingit

### CODE REVIEW

**Review by ayocodingit — APPROVED** [review]

File: core-service/src/domain/search.go [hunk 1]
```diff
@@ -30,12 +30,20 @@ type SearchListResponse struct {
 	CreatedAt string `json:"created_at" mapstructure:"created_at"`
 }
 
+// SuggestResponse ..
+type SuggestResponse struct {
+	ID    int    `json:"id"`
```
Comment (khihadysucahyo) [inline core-service/src/domain/search.go:35]: is the `id` property needed for this case?

File: core-service/src/domain/search.go [hunk 2]
```diff
@@ -30,12 +30,20 @@ type SearchListResponse struct {
...
+	Title string `json:"title"`
```
Comment (khihadysucahyo) [inline core-service/src/domain/search.go:36]: for global purposes for example we already have a future topic or context dictionary. I think it's less relevant to name it title(?)

File: core-service/src/domain/search.go (hunk 1 above)
Comment (rachadiannovansyah) [reply core-service/src/domain/search.go:35]: FE need this for unique props on element

File: core-service/src/domain/search.go (hunk 2 above)
Comment (rachadiannovansyah) [reply core-service/src/domain/search.go:36]: how about 'Topic'? or you have some insight?

**Review by firmanJS — APPROVED** [review]

File: core-service/src/domain/search.go (hunk 2 above)
Comment (rachadiannovansyah) [reply core-service/src/domain/search.go]: we deal with it https://github.com/jabardigitalservice/portal-jabar-services/pull/76/commits/ab495badc30d9f07bdba9002041c12eb2a3f61b5

reference on https://www.addsearch.com/docs/api/search-suggestions/

**Review by khihadysucahyo — APPROVED** [review]

**Pull request merged**